### PR TITLE
Create simple UI for Algobuilder - multisignature transaction

### PR DESCRIPTION
**Pull request by thdailong**
* Create UI for multisignature
* 
image

* Some file are need to make wallet connect components work
* Need more review to remove unused file

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
+{
+	"name": "sigmadao-ui",
+	"description": "Sigma DAO UI",
```
Comment (vuvoth) [inline package.json]: Please update package.json

File: src/api/constants.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const BACKEND_URL =
```
Comment (vuvoth) [inline src/api/constants.ts]: Please remote unused files.

File: src/contract/dao/dao-fund-lsig.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
+import { compileSignature } from "@/utility";
```
Comment (vuvoth) [inline src/contract/dao/dao-fund-lsig.ts]: Same here!

**Review by vuvoth — COMMENTED** [review]
Quick review! Please remove unused files and update package.json.

**Review by vuvoth — APPROVED** [review]
lgtm! Please ensure UI still work before merge!

File: .github/dependabot-example.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# version: 2
```
Comment (rmeena840) [inline .github/dependabot-example.yml:1]: GitHub action check is failing.

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -0,0 +1,53 @@
+name: ci
```
Comment (rmeena840) [inline .github/workflows/ci.yml:1]: Check this too.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+	"description": "Multisignature Sign UI",
+	"homepage": "https://algobuilder.dev",
+	"license": "mpl-2.0",
+	"engines": {
+		"node": ">=14.11.0",
+		"yarn": ">=3.1.0"
+	},
+	"scripts": {
+		"dev": "vue-cli-service serve",
+		"server": "nodemon backend/server.ts",
```
Comment (rmeena840) [inline package.json]: You don't need this server script.

File: package.json [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+		"algosdk": "^1.16.0",
+		"ant-design-vue": "^3.2.5",
+		"bootstrap": "^5.1.3",
+		"bootstrap-vue": "^2.22.0",
+		"core-js": "^3.21.1",
+		"cors": "^2.8.5",
+		"crypto-browserify": "^3.12.0",
+		"express": "^4.18.1",
+		"pinia": "^2.0.12",
+		"postgraphile": "^4.12.9",
```
Comment (rmeena840) [inline package.json]: You also don't need this. Please keep what is required in the project.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Please remove unnecessary files. Also run `yarn lint:fix` to fix the linter. I see many files that are of no use to this project.

File: package.json (hunk 6 above)
Comment (thdailong) [reply package.json]: The script "dev" we should keep to run the server too!

Comment (rmeena840) [reply package.json]: No, it is independent. You can remove server script.

Comment (thdailong) [reply package.json]: So I will remove `"server": "nodemon backend/server.ts"`

Comment (rmeena840) [reply package.json]: Yes

File: package.json [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+		"@vue/cli-service": "^5.0.1",
+		"@vue/compiler-sfc": "^3.2.31",
+		"@vue/eslint-config-typescript": "^10.0.0",
+		"eslint": "^8.11.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-vue": "^8.5.0",
+		"husky": "^7.0.4",
+		"jest": "^27.5.1",
+		"lint-staged": "^12.3.5",
+		"nodemon": "^2.0.16",
```
Comment (rmeena840) [inline package.json]: `nodemon, ts-node, cors` are also needed in this project. Please check this.
Also, you are using both ant design and bootstrap library. Do you need both?

File: src/config/algob.config.ts [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import algosdk from "algosdk";
```
Comment (rmeena840) [inline src/config/algob.config.ts]: You also don't need this file. Please check.

File: src/config/indexer.config.ts [hunk 10]
```diff
@@ -0,0 +1,16 @@
+import algosdk from "algosdk";
```
Comment (rmeena840) [inline src/config/indexer.config.ts]: You also don't need this file. Please check.

File: src/constants/constant.ts [hunk 11]
```diff
@@ -0,0 +1,21 @@
+export const GOV_TOKEN_ASSET = "gov-token";
```
Comment (rmeena840) [inline src/constants/constant.ts]: It seems like there are a few constants in this file which we are not using. Please remove not used constants.

File: src/constants/error.constant.ts [hunk 12]
```diff
@@ -0,0 +1,24 @@
+import { message, notification } from "ant-design-vue";
```
Comment (rmeena840) [inline src/constants/error.constant.ts:1]: Same in this constant file.

File: src/main.ts [hunk 13]
```diff
@@ -0,0 +1,11 @@
+import { createApp } from "vue";
+import App from "./App.vue";
+import { createPinia } from "pinia";
+import router from "./router";
+import Antd from "ant-design-vue";
+import "ant-design-vue/dist/antd.css";
+import "bootstrap/dist/css/bootstrap.css";
```
Comment (rmeena840) [inline src/main.ts]: Are we using bootstrap anywhere?

File: src/styles/layout.css [hunk 14]
```diff
@@ -0,0 +1,14 @@
+.flexbox {
```
Comment (rmeena840) [inline src/styles/layout.css]: I think this css is not used. Please check.

File: src/styles/style.css [hunk 15]
```diff
@@ -0,0 +1,32 @@
+@import url(./wallet.css);
```
Comment (rmeena840) [inline src/styles/style.css:1]: Check this css too.

File: src/types/interface.types.ts [hunk 16]
```diff
@@ -0,0 +1,12 @@
+import { NetworkTypes, WebModeTypes, WalletType } from "@/types";
+
+export interface WalletStoreState {
+	walletKind: WalletType;
+	webMode: WebModeTypes;
+	address: string;
+	network: NetworkTypes;
+}
+
+export interface UnknownObject {
```
Comment (rmeena840) [inline src/types/interface.types.ts]: I think we are not using this interface anywhere. Please check.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
There is still cleanup needed. I have commented on the file which I think is not relevant to the project. Please check constants and CSS are used anywhere. 
Also, you have also added bootstrap. Are we even using it anywhere?

File: src/constants/error.constant.ts (hunk 12 above)
Comment (thdailong) [reply src/constants/error.constant.ts:1]: This one is needed for the function `openErrorNotificationWithIcon`

File: src/styles/layout.css (hunk 14 above)
Comment (thdailong) [reply src/styles/layout.css]: Yes I have left only the class `.bottom_center`

File: src/styles/style.css (hunk 15 above)
Comment (thdailong) [reply src/styles/style.css:1]: I will used wallet.css for future purpose

File: package.json (hunk 8 above)
Comment (rmeena840) [reply package.json]: You were supposed to remove this dependency and check other.

File: package.json [hunk 17]
```diff
@@ -0,0 +1,62 @@
...
```
Comment (rmeena840) [inline package.json]: You were supposed to remove this dependency and check other.

File: src/config/algob.config.ts [hunk 18]
```diff
@@ -1,11 +1,7 @@
-import algosdk from "algosdk";
-
 const token =
 	process.env.ALGOSDK_TOKEN ||
```
Comment (rmeena840) [inline src/config/algob.config.ts:2]: This file is not needed entirely, i believe. Please check

**Comment by rmeena840** [comment]
@thdailong More cleanup is needed.

File: src/types/types.ts [hunk 19]
```diff
@@ -0,0 +1,16 @@
+import {
+	MyAlgoWalletSession,
+	WallectConnectSession,
+	WebMode,
+} from "@algo-builder/web";
+
+export type StateValue = string | number | bigint;
```
Comment (rmeena840) [inline src/types/types.ts]:
```suggestion

```

File: src/types/types.ts [hunk 20]
```diff
@@ -0,0 +1,16 @@
...
+
+export type Key = string;
```
Comment (rmeena840) [inline src/types/types.ts]:
```suggestion

```

File: src/types/types.ts [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+
+export type DateRange = "start" | "end";
```
Comment (rmeena840) [inline src/types/types.ts]:
```suggestion

```

File: package.json [hunk 22]
```diff
@@ -0,0 +1,62 @@
...
+		"test": "echo 'no test'"
+	},
+	"dependencies": {
+		"@algo-builder/algob": "^4.0.0",
+		"@algo-builder/web": "^4.0.0",
+		"@ant-design/icons-vue": "^6.1.0",
+		"algosdk": "^1.16.0",
+		"ant-design-vue": "^3.2.5",
+		"core-js": "^3.21.1",
+		"cors": "^2.8.5",
```
Comment (rmeena840) [inline package.json]:
```suggestion

```

File: package.json (hunk 17 above)
Comment (rmeena840) [inline package.json]:
```suggestion

```

File: package.json [hunk 23]
```diff
@@ -0,0 +1,62 @@
...
+		"@vue/eslint-config-typescript": "^10.0.0",
+		"eslint": "^8.11.0",
+		"eslint-config-prettier": "^8.5.0",
+		"eslint-plugin-vue": "^8.5.0",
+		"husky": "^7.0.4",
+		"jest": "^27.5.1",
+		"lint-staged": "^12.3.5",
+		"nodemon": "^2.0.16",
+		"prettier": "^2.5.1",
+		"ts-node": "^10.7.0",
```
Comment (rmeena840) [inline package.json]:
```suggestion

```

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Better now. Left few commit suggestions. Verify them.

File: package.json [hunk 24]
```diff
@@ -51,10 +50,6 @@
 		"eslint": "^8.11.0",
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-vue": "^8.5.0",
-		"husky": "^7.0.4",
```
Comment (rmeena840) [inline package.json]: @thdailong husky is needed.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviwed

File: package.json [hunk 25]
```diff
@@ -51,10 +50,6 @@
...
-		"jest": "^27.5.1",
-		"lint-staged": "^12.3.5",
-		"nodemon": "^2.0.16",
 		"prettier": "^2.5.1",
 		"ts-node": "^10.7.0",
```
Comment (rmeena840) [inline package.json]: We also dont need `ts-node`. Please check. I added in my earlier comment as well. `husky` is needed.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
Reviwed

File: package.json (hunk 24 above)
Comment (thdailong) [reply package.json]: oops, I delete too much

**Review by rmeena840 — APPROVED** [review]
lgtm now!!.

**Comment by robert-zaremba** [comment]
thanks for review @rmeena840

**Pull request merged**